uint32   timestamp
float32  velocity
float32  steering_angle